src/test/java/org/apache/commons/csv/issues/JiraCsv149Test.java

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
I added a different test to git master that does not contain so much code duplication. Please rebase on master and see my other comment.